build/docker/ubuntu-focal/Dockerfile

```diff
@@ -218,9 +218,15 @@ RUN apt-get install -y --no-install-recommends \
       php-dev \
       php-json \
       php-pear \
+      php-mbstring \
+      php-xml \
+      php-curl \
       re2c \
       composer
 
+RUN pecl install xdebug-3.1.1 && \
+      echo "zend_extension=xdebug.so" > /etc/php/7.4/cli/conf.d/20-xdebug.ini
```

Comment:
missing extensions and xdebug for some docker builds

Reply:
Looks like there may be an issue here.
The Dockerfile installs PHP without specifying a version, so it will install the latest version available in the package repository. However, the Xdebug configuration is written to a specific PHP 7.4 directory. If the installed PHP version is not 7.4, the Xdebug configuration will not be applied because PHP won't be able to find it.

Same for other Dockerfiles as well.

Reply:
It was easier than I thought :)

RUN apt-get install -y --no-install-recommends php-xdebug

Reply:
@pkvach will be updated here https://github.com/apache/thrift/pull/2937/commits/a8fb7267d3f52488263a834843fca8e66ec80fc3